libs/pages/application/src/lib/feature/page-settings-deployment-restrictions-feature/crud-modal-feature/crud-modal-feature.tsx

```diff
@@ -0,0 +1,61 @@
+import {
+  type ApplicationDeploymentRestriction,
+  DeploymentRestrictionModeEnum,
+  DeploymentRestrictionTypeEnum,
+} from 'qovery-typescript-axios'
+import { FormProvider, useForm } from 'react-hook-form'
+import { type ApplicationType } from '@qovery/domains/services/data-access'
+import { useCreateDeploymentRestriction, useEditDeploymentRestriction } from '@qovery/domains/services/feature'
+import CrudModal from '../../../ui/page-settings-deployment-restrictions/crud-modal/crud-modal'
+
+export interface CrudModalFeatureProps {
+  deploymentRestriction?: ApplicationDeploymentRestriction
+  onClose: () => void
+  serviceId: string
+  serviceType: ApplicationType
+}
+export function CrudModalFeature({ deploymentRestriction, serviceId, serviceType, onClose }: CrudModalFeatureProps) {
+  const serviceParams = {
+    serviceId,
+    serviceType,
+  }
+  const { mutate: createRestriction, isLoading: isCreateRestrictionLoading } =
```

Comment:
I have the feeling, the `isLoading` for the two mutates don't work, it's always "false", do you have an idea?

Reply:
![2023-08-25-154015_998x871_scrot](https://github.com/Qovery/console/assets/1716173/08977b6f-4ca0-4f68-bfcf-a0d62488286b)
It's not always false but you have to turn `mutate` into `mutateAsync` to see the loader. Otherwise it's too fast and you cannot put a breakpoint.